### PR TITLE
Update Calico and Canal to v3.13.2

### PR DESCRIPTION
**Pull request by hakman**
https://docs.projectcalico.org/v3.13/release-notes/#v3132

### CODE REVIEW

File: upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.16.yaml.template [hunk 1]
```diff
@@ -522,6 +529,10 @@ spec:
         k8s-app: calico-typha
         role.kubernetes.io/networking: "1"
       annotations:
+        # This, along with the CriticalAddonsOnly toleration below, marks the pod as a critical
+        # add-on, ensuring it gets priority scheduling and that its resources are reserved
+        # if it ever gets evicted.
+        scheduler.alpha.kubernetes.io/critical-pod: ''
```
Comment (KashifSaadat) [inline upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.16.yaml.template:535]: This annotation is removed from 1.16 onwards: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md#deprecations-and-removals

File: upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.16.yaml.template [hunk 2]
```diff
@@ -631,6 +642,12 @@ spec:
       labels:
         k8s-app: canal
         role.kubernetes.io/networking: "1"
+      annotations:
+        # This, along with the CriticalAddonsOnly toleration below,
+        # marks the pod as a critical add-on, ensuring it gets
+        # priority scheduling and that its resources are reserved
+        # if it ever gets evicted.
+        scheduler.alpha.kubernetes.io/critical-pod: ''
```
Comment (KashifSaadat) [inline upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.16.yaml.template:650]: Same here, may be worth submitting a PR to Calico to remove it from their templates.

File: upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.16.yaml.template [hunk 3]
```diff
@@ -780,7 +797,7 @@ spec:
             privileged: true
           resources:
             requests:
-              cpu: 250m
+              cpu: 90m
```
Comment (KashifSaadat) [inline upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.16.yaml.template:800]: This is still `250m` upstream as far as I can see, any reason to lower it?

File: upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.16.yaml.template (hunk 2 above)
Comment (hakman) [reply upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.16.yaml.template]: True. Removed.

File: upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.16.yaml.template (hunk 3 above)
Comment (hakman) [reply upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.16.yaml.template:790]: https://github.com/kubernetes/kops/commit/ba76db7f41e7584d156f845197e68308a9050895

File: upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.16.yaml.template (hunk 2 above)
Comment (hakman) [reply upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.16.yaml.template]: Probably a good idea to submit a PR there. There are a few more cosmetic things that should be fixed.

File: upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.16.yaml.template (hunk 3 above)
Comment (KashifSaadat) [reply upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.16.yaml.template:790]: Aha thanks for linking, okay sounds good 👍

**Comment by hakman** [comment]
Thanks for the review @KashifSaadat :)

**Pull request merged**